service/qos/service_level_controller.hh

```diff
@@ -90,6 +90,9 @@ private:
     service_level _default_service_level;
     service_level_distributed_data_accessor_ptr _sl_data_accessor;
     sharded<auth::service>& _auth_service;
+    std::chrono::time_point<std::chrono::steady_clock> _last_successful_config_update;
+    int _logged_intervals;
```

Comment:
nitpick: the clock is monotonic, so technically you could use `unsigned` here. But if it causes annoying compilation warning during comparisons, I suppose it can be left as is

Reply:
ditto